project/Settings.scala

```diff
@@ -191,9 +191,7 @@ object Settings {
               s"-Dlogback.configurationFile=${(Test / baseDirectory).value.getAbsolutePath}/src/test/resources/logback-test.xml",
               s"-Djava.util.logging.config.file=${(Test / baseDirectory).value.getAbsolutePath}/src/test/resources/logback-test.xml",
               s"-Dtest.name=${test.name}",
-              s"-Ddir.name=${(Test / baseDirectory).value}",
-              s"-Dheadless=${Option(System.getProperty("headless")).getOrElse("false")}",
-              s"-Djsse.enableSNIExtension=${Option(System.getProperty("jsse.enableSNIExtension")).getOrElse("false")}"))
```

Comment:
`-Dheadless` and `-Djsse.enableSNIExtention` were breaking automation tests on Java 11. Removing them seems to work as intended.